### PR TITLE
Solved the Alignment Issue with Navbar

### PR DESCRIPTION
**Pull request by ConnectBhawna**
## Description
Currently, the Navbar appears to be misaligned, lacking appropriate padding between the IHR Logo, Search bar, and other components. As a result, it looks unappealing and unprofessional on both desktop and mobile views. To improve the visual appearance of the Navbar, it is necessary to adjust the padding and ensure that all components are properly aligned.
I solved the issue #278

## Motivation and Context
In order to improve the overall user experience and ensure consistency throughout the website, it is essential to address the misaligned Navbar that is currently present. This Navbar lacks the necessary padding between the IHR Logo, Search bar, and other components, resulting in an unappealing and unprofessional appearance on both desktop and mobile views.

To address this issue, adjustments must be made to the padding and alignment of all components within the Navbar. Appropriate padding must be added to ensure that each component is clearly defined and separated from one another, while also maintaining a cohesive design. Additionally, it is crucial to ensure that all elements within the Navbar are properly aligned to create a visually pleasing and user-friendly experience.

By making these changes to the Navbar, the overall appearance and usability of the website can be significantly improved, resulting in a more professional and consistent user experience.

## How Has This Been Tested?
As part of my testing process, I decided to create a brief video showcasing the website as it runs on a local system. In my opinion, the video serves as a useful tool for evaluating the site's performance and identifying any areas that may require improvement. Additionally, I find the video to be visually impressive and engaging.

Here is the short video Link :
[Navbar.webm](https://user-images.githubusercontent.com/81790585/221372032-b2f799cb-6fbd-46f6-a32a-78d5eafc3c09.webm)

## Screenshots :
Prior to the bug being fixed, the Navbar exhibited an issue that can be clearly seen in the provided screenshot. This issue may have included misaligned or overlapping components, inadequate padding, or other inconsistencies that resulted in an unappealing or unprofessional appearance.

It is important to address such issues as they can negatively impact the overall user experience, causing frustration and confusion. By identifying and resolving the issue in the Navbar, the website can maintain its credibility and professionalism, as well as improve user engagement and satisfaction.

<table>
  <tr>
    <td>Before the error fixed</td>
     <td>After the error fixed</td>
  </tr>
  <tr>
    <td></td>
     <td></td>
    
  </tr>
</table>

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/views/Header.vue [hunk 1]
```diff
@@ -1,40 +1,40 @@
...
-                  </q-item-section>
-                </q-item>
-              </q-list>
-            </q-btn-dropdown>
-          </q-btn-group>
-        </div>
-        <div class="lt-md">
-          <q-btn flat dense round @click="leftDrawerOpen = !leftDrawerOpen" icon="menu" aria-label="Menu"></q-btn>
-        </div>
+      <div class="col-12 row no-wrap items-center justify-between">
```
Comment (thisiskaransgit) [inline src/views/Header.vue:4]: Hey @ConnectBhawna, thanks for this, even though it's a small change but it affects the user experience, 
right now the elements are far apart, rather distribute them evenly 
```suggestion
      <div class="col-12 row no-wrap items-center justify-around">
```

**Comment by thisiskaransgit** [comment]
Also kindly, revert the formatting changes that you implemented

File: src/views/Header.vue (hunk 1 above)
Comment (ConnectBhawna) [reply src/views/Header.vue:4]: Okk so what i need to do make it  justify-between?

**Comment by ConnectBhawna** [comment]
Also can you please ellobrate which formatting changes you are discussing here?

File: src/views/Header.vue (hunk 1 above)
Comment (thisiskaransgit) [reply src/views/Header.vue:4]: `justify-between` is  separating elements far apart, change it to `justify-around` so that it gets distributed evenly

**Comment by thisiskaransgit** [comment]
The extra space that you added in the source code from line 5 to line 37

File: src/views/Header.vue (hunk 1 above)
Comment (ConnectBhawna) [reply src/views/Header.vue:4]: Ohhh got it
Thanks @thisiskaransgit for helping me out in this. I will make do this by today :)

**Comment by Aniket762** [comment]
Hey @ConnectBhawna which issue are you solving? Sorry, but I was not able to find the issue related to the navbar styling.

**Comment by ConnectBhawna** [comment]
Sure @Aniket762  actually i haven't created the issue, I directly make the PR for the same.
Can you please look into it hope it works :)

**Comment by romain-fontugne** [comment]
hmmm, looking at the two screenshots I'm not sure which one I prefer... The github website has the logo, seach bar, and nav bar to the left (like we have currently on ihr). Github also has user button on the right which I think make it well balanced.

Eventually we'll have user login/ settings on IHR too, so I'm not sure what we should do

**Comment by ConnectBhawna** [comment]
According to me sir,
Currently the Navbar design not looking that much good in Mobile as well as in Desktop view. As you can see .
Screenshot from 2023-02-26 18-56-17
According to me this one looks more good in both Mobile and desktop view..
Let me know what works for you :)

**Comment by romain-fontugne** [comment]
well, then I think the problem is mainly for small screens? It may be better to have the search bar taking all space for small screens?

**Comment by ConnectBhawna** [comment]
nope sir for both desktop and mobile view :)

**Comment by ConnectBhawna** [comment]
According to me , it doesn't looks good in both small and desktop screen. As it leave lots of space in the right and no space in the left, also no space between the search bar, logo and navar components.

So i try to solve this part like this.
For desptop view :
Screenshot from 2023-02-27 17-57-47

For Small screen :
Screenshot from 2023-02-27 17-58-47

So as you can see all the components are eventually distributed with the equal space. It looks more formal.
Let me know your view point

**Comment by Aniket762** [comment]
Hey @ConnectBhawna, closing this issue. This is something we are not looking to solve immediately and since we are looking for a revamp we will consider solving the CSS issues later.

closes #278

**Pull request closed without merging**